bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/contentfragment/AbstractContentFragmentDataSourceServlet.java

```diff
@@ -26,60 +25,38 @@
 import org.apache.sling.api.SlingHttpServletRequest;
 import org.apache.sling.api.SlingHttpServletResponse;
 import org.apache.sling.api.resource.Resource;
-import org.apache.sling.api.resource.ResourceMetadata;
 import org.apache.sling.api.resource.ResourceResolver;
 import org.apache.sling.api.resource.ValueMap;
-import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
-import org.apache.sling.api.wrappers.ValueMapDecorator;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 import com.adobe.cq.dam.cfm.ContentFragment;
 import com.adobe.granite.ui.components.Config;
-import com.adobe.granite.ui.components.ExpressionHelper;
-import com.adobe.granite.ui.components.ExpressionResolver;
 import com.adobe.granite.ui.components.ds.DataSource;
 import com.adobe.granite.ui.components.ds.EmptyDataSource;
 import com.adobe.granite.ui.components.ds.SimpleDataSource;
-import com.adobe.granite.ui.components.ds.ValueMapResource;
 
 import static com.adobe.cq.wcm.core.components.models.contentfragment.ContentFragment.PN_PATH;
-import static org.apache.sling.api.resource.Resource.RESOURCE_TYPE_NON_EXISTING;
+
 
 /**
  * <p>Abstract datasource providing access to a content fragment instance configured by datasource properties. The
  * content fragment to be returned by {@link #getContentFragment(SlingHttpServletRequest)} can be specified via a
- * {@code /apps/core/wcm/components/contentfragment} component (see {@link #PN_COMPONENT_PATH}) or
+ * {@code /apps/core/wcm/extension/components/contentfragment} component (see {@link #PN_COMPONENT_PATH}) or
```

Comment:
Shouldn't `extension` be removed from the path?

Reply:
Probably a merge error. Will fix this.